bluesky/tests/test_persistent_dict.py

```diff
@@ -0,0 +1,34 @@
+import collections.abc
+
+from bluesky.utils import PersistentDict
+import numpy
+from numpy.testing import assert_array_equal
```

Comment:
Would you mind to use the sorting approach for the imports we discuss before (such as native Python libs go first, then community packages, and then our development)?